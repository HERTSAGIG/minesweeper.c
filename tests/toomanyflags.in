g 9 9
b 1 1
b 2 2
b 3 3
b 4 4
b 5 5
b 6 6
b 7 7
b 8 8
b 1 2
b 1 3
f 0 0
f 0 1
f 1 0
f 1 1
f 2 0
f 2 1
f 2 2
f 3 0
f 3 1
f 3 2
f 4 0
f 4 1
