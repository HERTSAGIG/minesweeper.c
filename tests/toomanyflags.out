g 9 9
b 1 1
b 2 2
b 3 3
b 4 4
b 5 5
b 6 6
b 7 7
b 8 8
b 1 2
b 1 3
+---------+
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 0 0
+---------+
|f********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 0 1
+---------+
|f********|
|f********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 1 0
+---------+
|ff*******|
|f********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 1 1
+---------+
|ff*******|
|ff*******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 2 0
+---------+
|fff******|
|ff*******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 2 1
+---------+
|fff******|
|fff******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 2 2
+---------+
|fff******|
|fff******|
|**f******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 3 0
+---------+
|ffff*****|
|fff******|
|**f******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 3 1
+---------+
|ffff*****|
|ffff*****|
|**f******|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
f 3 2
+---------+
|ffff*****|
|ffff*****|
|**ff*****|
|*********|
|*********|
|*********|
|*********|
|*********|
|*********|
+---------+
error
